internals/overlord/servstate/handlers.go

```diff
@@ -114,11 +114,8 @@ func (m *ServiceManager) doStart(task *state.Task, tomb *tomb.Tomb) error {
 		return err
 	}
 
-	mgrPlan, err := m.Plan()
-	if err != nil {
-		return fmt.Errorf("cannot fetch plan: %w", err)
-	}
-	config, ok := mgrPlan.Services[request.Name]
+	planSnapshot := m.plan()
```

Comment:
I made it ```currentPlan```. Could that work for you? Since the ```internals/plan``` package is already present / imported, i've tried to not use plan directly as well.

Reply:
Yep, `currentPlan` is fine, thanks.